data/.wrglsync.yaml

```diff
@@ -72,6 +74,8 @@ repos:
     fileName: match/cprr_plaquemines_so_2019.csv
   match-lprr-louisiana-state-csc-1991-2020:
     fileName: match/lprr_louisiana_state_csc_1991_2020.csv
+  match-cprr-scott-pd-2020:
+    fileName: match/cprr_scott_pd_2020.csv
```

Comment:
Going forward perhaps we don't need to keep clean and match script in WRGL. Doesn't look like anybody is interested in how those files change.

Reply:
Let's discuss. Why do you think we should not? I think it's helpful to keep it all in one place.

Reply:
Maybe I'm wrong but I just feel like we don't have any eyeballs on those files. But we can keep them if you're in doubt.